crates/kitsune_p2p/kitsune_p2p/src/gossip/simple_bloom/step_4_com_loop_inner.rs

```diff
@@ -157,54 +157,49 @@ pub(crate) async fn step_4_com_loop_inner_incoming(
             );
 
             // parse/integrate the chunks
-            let futs =
-                bloom.inner.share_mut(move |i, _| {
-                    if let Some(endpoint) = i.initiate_tgt.clone() {
-                        if finished && con.peer_cert() == *endpoint.cert() {
-                            i.initiate_tgt = None;
-                        }
+            let futs = bloom.inner.share_mut(move |i, _| {
```

Comment:
Are we ready to delete simple_bloom yet? If everybody's switched over to the default sharded_gossip, we shouldn't spend cycles maintaining something that isn't needed.

Reply:
Yeh sounds like a good idea, maybe in a separate PR?